packages/client/components/MeetingsDashRoot.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import graphql from 'babel-plugin-relay/macro'
+import {QueryRenderer} from 'react-relay'
+import withAtmosphere, {
+  WithAtmosphereProps
+} from '../decorators/withAtmosphere/withAtmosphere'
+import renderQuery from '../utils/relay/renderQuery'
+import MeetingsDash from './MeetingsDash'
+import {LoaderSize} from '../types/constEnums'
+
+const query = graphql`
+  query MeetingsDashRootQuery {
+    viewer {
+      ...MeetingsDash_viewer
+    }
+  }
+`
+
+interface Props extends WithAtmosphereProps { }
+
+const MeetingsDashRoot = (props: Props) => {
```

Comment:
Looking for feedback from a friendly Parabol developer. Is this a good way to go about this? To have a root query for the meetings view?

Reply:
for now, I'd say it's not necessary & just use a fragment.
reason being:
- an extra query means more waterfall loading (the previous query has to load before this query can be fetched, so it'll take longer)
- we're already fetching most of these fields for the active meetings notification, so the amount of extra info isn't too great

Reply:
The problem that I have is that I’m setting up a new route in `Dashboard.tsx` for `/meetings` — all of the routes have a route component. I’m not sure how to pass along the `viewer` through the router.